chrome/content/zotero-platform/mac/overlay.css

```diff
@@ -6,15 +6,6 @@
 	-moz-binding: none !important;
 }
 
-.zotero-item-pane-content .groupbox-body {
-	-moz-appearance: none;
-	background-color: #ffffff;
-}
-
-#zotero-item-pane-message-box description {
-	color: #7f7f7f;
-}
```

Comment:
Why was this removed? I'm not sure why this was Mac-specific, but if anything it should be moved to a global style.

Reply:
I didn't see this in the figma so currently this is replaced by the global style to `fill-tertiary`

Reply:
It's https://github.com/windingwind/zotero/blob/4dc5a2c3c280c77e15d04a24e79a5cb2bf111026/scss/elements/_itemDetails.scss#L3